docs/user/00-25-plug-in-istio-ca.md

```diff
@@ -0,0 +1,44 @@
+# Configure Istio CA with Custom Certificates
+For enhanced security, replace Istio's default self-signed certificates with administrator-provided certificates.
+
+## Default Istio Behavior
+
+By default, Istio generates a self-signed root certificate and uses it to sign certificates for each workload in the service mesh.
+
+Istio follows a standard PKI certificate hierarchy:
+
+- **Root CA**: Self-signed top-level certificate that issues intermediate certificates. It's recommended that this certificate runs on an offline secure machine.
+- **Intermediate CAs**: Issued by the Root CA to each Istio cluster.
+- **Workload Certificates**: Automatically generated by Istio for each Pod and signed by the cluster's intermediate CA.
+
+### Configuring a Custom CA
+
+You can replace the default self-signed certificate provided by Istio and use a certificate recommended by your organization.
+
+To configure the certificate in Kyma, in each cluster's `istio-system` namespace, you must create a Secret `cacerts`. The Secret includes the following input files:
+
+| File | Description |
+|------|-------------|
+| `ca-cert.pem` | The intermediate CA certificate with permissions to sign server certificates. You can use any CA certificate in your organization's chain of trust.
+| `ca-key.pem` | The private key corresponding to `ca-cert.pem`. Istio uses it to sign new workload certificates. |
+| `root-cert.pem` | The top-level root CA certificate in your trust chain.
+| `cert-chain.pem` | The complete certificate chain from your intermediate CA (`ca-cert.pem`) to the root CA (`root-cert.pem`).
+
+
+To create the secret for cluster `cluster1`, run:
+
+```bash
+kubectl create secret generic cacerts -n istio-system \
+    --from-file=cluster1/ca-cert.pem \
+    --from-file=cluster1/ca-key.pem \
+    --from-file=cluster1/root-cert.pem \
+    --from-file=cluster1/cert-chain.pem
+```
+
+For more information, see [Plug in CA Certificates](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/).
+
+### Security Considerations
+
+For enhanced security, it is not recommended that you manage sensitive data directly in a Kyma cluster. Instead, use a production-ready CA and manage your root CA on a secure, offline machine.
```

Comment:
```suggestion
For live landscapes it is recommended to use a production-ready CA and manage your root CA on a secure, offline machine. In addition it might be worth to think about an additional intermediate CA outside of the cluster and automate the submission of the CA certificates to the clusters.
```